src/Block/FilterVars.php

```diff
@@ -38,16 +44,24 @@ public function __construct(
      * Fetch document view
      *
      * @return string
+     * @throws ContextNotFoundException
```

Comment:
I don't see you throwing this. Can we maybe not document throws that happen deeper within the code?

One can prevent this from being marked by their IDE by changing the call tree analysis depth to `0`:

![image](https://user-images.githubusercontent.com/642417/103887535-7bd09e80-50e3-11eb-9bf9-5e5577143540.png)

Reply:
We can propose this as an update to the settings injected by the testing suite.